### PR TITLE
Hui animations

### PR DESCRIPTION
**Pull request by incredibleLeitman**
implemented some animations for ui and a wonky sound effect if no weapon ammo is remaining when the payer tries to shoot

### CODE REVIEW

File: recursio-client/Characters/Player.gd [hunk 1]
```diff
@@ -76,6 +77,12 @@ func apply_input(movement_vector: Vector3, rotation_vector: Vector3, buttons: in
 		if action.ammunition > 0:
 			_aim_visuals.visible = true
 			_aim_visuals.get_child(timeline_index % 2).visible = aim_mode
+		else:
+			# QUESTION: should this be in ClientActionManager?
```
Comment (kb173) [inline recursio-client/Characters/Player.gd:81]: No I think this is fine :+1:

**Review by kb173 — APPROVED** [review]
looks good! would be good if @AmonShokhinAhmed could look it over for the colors too though.
I saw some hardcoded colors like `Color.red`, maybe it'd be good to move all of that to the Constants for easier configuration, but since it's mainly used for modulation I don't think it's 100% necessary.

File: recursio-client/Characters/Player.gd (hunk 1 above)
Comment (AmonShokhinAhmed) [reply recursio-client/Characters/Player.gd:81]: Also think this is fine, because the Player script is client side only as well.

**Review by AmonShokhinAhmed — APPROVED** [review]
Very cool. I fixed hard coded colors with 46fbfa4 and removed question comment with 8174afc.  I think this looks good now.

**Pull request merged**